libs/core/libs/shared/src/core_ops.rs

```diff
@@ -28,7 +28,7 @@ where
     T: TreeLike<F = SignedFile>,
 {
     // todo: revisit logic for what files can be finalized and how e.g. link substitutions, deleted files, files in pending shares, linked files
-    pub fn finalize(
+    fn finalize(
```

Comment:
while we're here, I'm down to rename this to anything else, rename `resolve_and_finalize` to `finalize`, and rename `resolve_and_finalize_all` to `finalize_all`